docs/operations/metrics.md

```diff
@@ -0,0 +1,26 @@
+# Getting prod metrics for EASi
+
+You'll likely need to fetch metrics for us at some point for the
+project to understand the impact EASi is making.
+
+## Steps
+
+1) Visit [EASi Production](https://cms.easi.gov)
+2) Login with your EUA credentials
+3) Open up the network tab in your browser
+
+    - Look for a request that hits our API
+(the request URL in the headers section will
+include `https://cms.easi.gov/api/v1`)
+    - Look for the request header `Authorization`
```

Comment:
Can we just get this from local storage instead? That should be the source of truth.

Reply:
yep, I was trying to make it easier for non-devs to get this value